src/shared/SellerProfile/Body.js

```diff
@@ -66,46 +75,90 @@ const Body = (props) => {
         })}
         </ul>
       </Row>
-
+        <Row title="For opportunities" show={true}>
+        <p>
+            <b>Contact</b><br/>
+            <span>{contact_name}</span>
+        </p>
+        <p>
+            <b>Phone</b><br/>
+            <span>{contact_phone}</span>
+        </p>
+        <p>
+            <b>Email</b><br/>
+            <a href={`mailto:${contact_email}`}>{contact_email}</a>
+        </p>
+        </Row>
       <Row title="Company Details" show={true}>
         <b>Business Representative</b><br/>
-        <p>{representative}</p>
+          <p>
+              <span>{representative}</span><br/>
+              { email && <span><a href={`mailto:${email}`}>{email}</a><br/></span>}
+              { phone && <span>{phone}<br/></span>}
+          </p>
+        <p></p>
 
         <b>Website</b><br/>
-        <p><a href={website} rel="external">{website}</a></p>
+        <p><a href={website} rel="external" target="_blank">{website}</a></p>
 
-        <b>Linkedin Profile</b><br/>
-        <p><a href={linkedin} rel="external">{linkedin}</a></p>
+          { linkedin &&<span><b>Linkedin Profile</b><br/>
+           <p><a href={linkedin} rel="external">{linkedin}</a></p></span> }
 
         <b>Main Address</b><br/>
         <p>
           <span>{address.address_line}</span><br/>
           <span>{address.suburb}</span><br/>
           <span>{address.state} {address.postal_code}</span>
         </p>
-
+          { interstate && <p><b>This seller is able work interstate</b></p>}
         <b>ABN</b><br/>
         <p>{abn}</p>
       </Row>
+      <Row title="Documents" show={Object.keys(documents).length}>
+          {Object.keys(documents).map((key, val) =>
+              <p key={val}>
+
+                  <a href={`${documentsUrl}${documents[key]}`}><Switch>
```

Comment:
could have done this without switch eg.  have another file that maps key to string
{
   financial: 'Financial Statement'
}